### PR TITLE
Type of Item field #516

### PR DESCRIPTION
**Pull request by criedlberger**

### CODE REVIEW

File: config/initializers/sufia.rb [hunk 1]
```diff
@@ -89,8 +89,10 @@
   # please run rake db:seed to create the collections and restart httpd. 
   # The collection IDs will be added here
   # In production it assumes that the collections will be available in the system at the time of deposit
-  # config.cstr_collection_id = ""
-  # config.ser_collection_id = ""
+
+
+  config.cstr_collection_id = "q237hr920"
```
Comment (pbinkley) [inline config/initializers/sufia.rb]: Don't merge these !!!!!!

File: config/initializers/sufia.rb [hunk 2]
```diff
@@ -27,8 +27,8 @@
   config.resource_types = {
     "Book" => "Book",
     "Book Chapter" => "Book Chapter",
-    "Conference\/Workshop Poster" => "Conference\/Workshop Poster",
-    "Conference\/Workshop Presentation" => "Conference\/Workshop Presentation",
+    "Conference\/workshop Poster" => "Conference\/workshop Poster",
```
Comment (pbinkley) [inline config/initializers/sufia.rb]: We don't want to end up with "workshop Poster", do we? Not sure what's the intention here.

File: spec/features/admin_resource_types_spec.rb [hunk 3]
```diff
@@ -40,7 +40,7 @@
     end
 
     it 'has admin resource list' do
-      expect(page).to have_select('generic_file_resource_type', :options => ["Book","Book Chapter", "Computing Science Technical Report", "Conference/Workshop Poster","Conference/Workshop Presentation", "Dataset", "Image", "Journal Article (Draft-Submitted)", "Journal Article (Published)", "Learning Object", "Report", "Research Material", "Review", "Structural Engineering Report", "Thesis"])
+      expect(page).to have_select('generic_file_resource_type', :options => ["Book","Book Chapter", "Computing Science Technical Report", "Conference/workshop Poster","Conference/workshop Presentation", "Dataset", "Image", "Journal Article (Draft-Submitted)", "Journal Article (Published)", "Learning Object", "Report", "Research Material", "Review", "Structural Engineering Report", "Thesis"])
```
Comment (pbinkley) [inline spec/features/admin_resource_types_spec.rb]: Likewise. Is this an unintended search-and-replace?

File: config/initializers/sufia.rb (hunk 2 above)
Comment (weiweishi) [inline config/initializers/sufia.rb]: It's based on https://github.com/ualbertalib/HydraNorth/issues/503. There's inconsistency between hydranorth data dictionary and the deposit form. And Conference/workshop Poster and Conference/workshop Presentation is used in the current ERA and data dictionary.

Comment (pbinkley) [inline config/initializers/sufia.rb]: Ah, OK.

**Pull request merged**